fact_extractor/plugins/unpacking/jffs2/code/jffs2.py

```diff
@@ -18,8 +18,8 @@ def unpack_function(file_path, tmp_dir):
     local_tmp_dir should be used to store the extracted files.
     """
 
-    extract_dir = Path(tmp_dir) / 'jffs-root'
-    output = execute_shell_command(f'fakeroot jefferson -v -d {extract_dir} {file_path}') + '\n'
+    tmp_dir_path = Path(tmp_dir)
+    output = execute_shell_command(f'fakeroot jefferson -vfd {tmp_dir_path} {file_path}')
```

Comment:
Is the -f option the one doing the heavy lifting here? Or is it an artifact and the path renaming is the one responsible?

Reply:
not using the -f option was the reason the additional folder was previously necessary: jefferson refuses to work if the output path already exists -- except you use the `--force`/`-f` option.